src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx

```diff
@@ -15,22 +15,76 @@ redirects:
   - /docs/apm/apm-ui-pages/monitoring/apm-data-infrastructure
 ---
 
-import infrastructureInfraApmDataChart from 'images/infrastructure_screenshot-crop_infra-apm-data-chart.webp'
+import apmCriticalViolationsinApdex from 'images/apm_screenshot-full_critical-violations-in-Apdex.webp'
+
+import apmCPUPercentageViolationinAPM from 'images/apm_screenshot-crop_CPU-percentage-violation-in-APM.webp'
+
+import apmToggleBetweenHistogramViews from 'images/apm_screenshot-crop_toggle-between-histogram-views.webp'
+
+import apmExploringaSpecificHostinAPM from 'images/apm_screenshot-full_exploring-a-specific-host-in-APM.gif'
+
+import apmFilterbyApplication from 'images/apm_screenshot-full_filter-by-application.webp'
 
 The integration of [APM](/docs/apm/new-relic-apm/getting-started/welcome-new-relic-apm) and [infrastructure](/docs/infrastructure/new-relic-infrastructure/getting-started/welcome-new-relic-infrastructure) data lets you see your APM data and infrastructure data side by side so you can find the root cause of problems more quickly.
 
-The main ways to find and use APM data in infrastructure monitoring are:
+First, this doc will walk you through the process of resolving infrastructure issues with APM. Then it will dig deeper into some of the key features of APM and infrastructure monitoring. 
 
-* [View APM charts on infrastructure monitoring UI pages](#apm-chart)
-* [Filter hosts by application data](#filter-by-app)
-* [Switch between infrastructure and APM](#switch)
-* [Examine APM data in Inventory and Events pages](#inventory-events)
+<Steps>
+    <Step>
+    ## Do you have a problem in your system? 
 
-Infrastructure data appears in <InlinePopover type="apm" /> in the host table on the [APM Summary](/docs/apm/applications-menu/monitoring/apm-overview-page) page.
+    You're the engineer responsible for the `Billing Service` application and you get an alert that says, "Error percentage > 45% for at least five minutes on `Billing Service`." The first thing you're going to do is go to the `Billing Service` application in APM and open the **Summary** page to get an overview of the health of your system. The first indicator of a problem in your system is a high Apdex score, which is a measure user satisfaction. Here you can see that the score is .79 and has triggered a critical violation. Next you're going to check your error rate. Here you can see that the error rate has hit 100%. 
```

Comment:
I'd break this up, maybe using bulleted steps to separate the actions that an eng/dev might take here. I also think you could add the preamble/context situation *before* the steps, making it clear that the stepped tutorial is walking User through an example use case

Reply:
sounds good! Thanks.